pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,111 @@
+
+[enterprise]
+### Security
+[/enterprise]
+- Secrets Management Service
+  - Binary Secret files are now supported
+  - Hierarchical access control is now supported.
+
+### Monitoring
+- The DC/OS metrics component now produces metrics in [Prometheus](https://prometheus.io/docs/instrumenting/exposition_formats/) format. [View the documentation](/1.11/metrics).
+- Unified logging API that provides simple access to container (task) as well as system component logs. [View the documentation](/1.11/monitoring/logging/logging-api/logging-v2/).
+
+### Storage
+- DC/OS Storage Service 0.1 (beta) - DSS users will be able to dynamically create volumes based upon profiles or policies to fine-tune their applications' storage requirements. This feature leverages the industry-standard Container Storage Interface (CSI) to streamline the development of storage features in DC/OS by Mesosphere and our community and partner ecosystems. [View the documentation](https://docs.mesosphere.com/services/beta-storage/0.1.0-beta/).[beta type="inline" size="small" /][enterprise type="inline" size="small" /]
+- Pods now support persistent volumes. [View the documentation](/1.11/deploying-services/pods).[beta type="inline" size="small" /]
+
+**Note:** Because these storage features are beta in 1.11, they must be explicitly enabled. Beta features are not recommended for production usage, but are a good indication of the direction the project is headed.
+
+### Updated DC/OS Data Services
+- TLS encryption for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS is now supported.
+- Fault domain awareness for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic and DC/OS HDFS. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed.
+- New API endpoint to pause a node for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. Use this endpoint to relaunch a node in an idle command state for debugging purposes.
+- New DC/OS Kafka ZooKeeper service. [View the documentation](/services/kafka-zookeeper).
+- You can now select a DC/OS data service version from a dropdown menu in the DC/OS UI.
+- Improved scalability for all DC/OS data services.
+
+## <a name="issues-fixed"></a>Improvements and Issues Fixed in DC/OS 1.11.1 Release
+
+- INFINITY-3331 - Fixed cleaning up other framework's volumes.
+- DCOS_OSS-2292 - Fixed crashes on empty lines for DC/OS CLI.
+- DCOS_OSS-2255 - Fixed dcos-installer-ui package failure to build.
+- DCOS_OSS-2247 - Fixed fatal error on checks that exceeded timeout on dcos-diagnostic service.
+- DCOS_OSS-2223 - Added encoders to support package registry. Fixed comsos to initialize empty objects for optional json elements from json schema.
+- DCOS_OSS-2210 - Ensured that cache timestamps are well-formed.
+- DCOS_OSS-2184 - Removed an invalid dns record for marathon.mesos.
```

Comment:
Consolidated the situation when two DC/OS services would compete for the same DNS A record

(I could not find the patch details in JIRA, so I had a hard time describing the actual solution)

CC @asridharan

Reply:
Removed DCOS_OSS-2184  based on Avinash's feedback below:
<img width="837" alt="screen shot 2018-04-17 at 1 35 27 pm" src="https://user-images.githubusercontent.com/36425832/38895208-3dba691a-4244-11e8-84b9-76566274b627.png">

@Anitha Muthu https://jira.mesosphere.com/browse/DCOS_OSS-2184 shouldn’t be part of the release notes. We didn’t actually do anything for this change. It was a misc-configuration by the user.